deploy-disconnected-registry/ocp-sync.sh

```diff
@@ -44,8 +44,16 @@ function mirror_ocp() {
 
     ####### WORKAROUND: Newer versions of podman/buildah try to set overlayfs mount options when
     ####### using the vfs driver, and this causes errors.
```

Comment:
nit: Seems that we're removing the change of the storage driver we were setting in the workaround, should the comments be removed too? (and have them in the function for the registry login?)

Reply:
i just moved it without the comments 

```shell
# shared-utils/common.sh
function registry_login() {
    # workaround runnning podman commands
    export STORAGE_DRIVER=vfs
    sed -i '/^mountopt =.*/d' /etc/containers/storage.conf

    ${PODMAN_LOGIN_CMD} ${1} -u ${REG_US} -p ${REG_PASS} --authfile=${PULL_SECRET}
    ${PODMAN_LOGIN_CMD} ${1} -u ${REG_US} -p ${REG_PASS}
}

```

Reply:
will move the comment too